docs/Project-Board-Guide.md

```diff
@@ -22,9 +22,9 @@ NB there is further documentation colocated with relevant code. The above list i
 <!-- prettier-ignore -->
 | Column Name               | Issues/Pull Requests that belong in this column | When to move card to the next column | Additional context |
 | ------------------------- | ------------------------------------------------| ------------------------------------ |------------------- |
-| To-do | Contains issues that have been newly created and added to the Simorgh project. | Once the issue has been refined by the corresponding labelled pod (aka team), then it can be manually moved to "Ready for Dev". | n/a |
+| Coming soon | Contains issues that we are going to work on within the next couple of weeks. | Once the issue has been refined by a World Service team, then it can be manually moved to "Backlog (Ready for Dev)". | n/a |
```

Comment:
```suggestion
| Coming Soon | Contains issues that we are going to work on within the next couple of weeks. | Once the issue has been refined by a World Service team, then it can be manually moved to "Backlog (Ready for Dev)". | n/a |
```